### DISCUSSION TITLE
Openrc service

### DISCUSSION THREAD

**Discussion by er2de2** [anywherelan/awl#99] [Q&A] [answered]
Hi.
I want run awl on my alpine linux vps server.
Can you make openrc service config file to run awl on system boot?

**Comment by pymq** [comment, marked as answer]
Unfortunately I'm not famillar with OpenRC or Alpine in general

Found these docs, maybe it'll help someone

https://docs.alpinelinux.org/user-handbook/0.1a/Working/openrc.html
https://github.com/OpenRC/openrc/blob/master/service-script-guide.md
https://wiki.gentoo.org/wiki/OpenRC/User_services
